config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+flhsLfBuTMe+6sezqHvHt8mOGXzUnNZDxRBodExQY56oAlMeMd4vx/U8gz1DuGLQCglPDnFolcXqoP81eS6tdfuALNvXkuG1DY/3TuejuXjWYju+pi5QzeBV6kT7hI2eRymDxixERfRUHQBDSiD8pR/v282DH4u6idqInuJJHjHycI9c0Nce6Ucjcf4U4UhYhgFqhJ7FB02kZLWMXIRbOruGzUVrK98vO4bAGwipNzwc5iUTICH0rDLejXrIepzHcsslOJ4cyf6CFCuyOAJOencWn4+BRVGcgZlxjA6uKP7/pJebq/4kSIcHskkAfhIkabXI9Rc4nsvqadzCGhkOMG/1gpuO7on+E3ZlhMcFCTSZpbuU3NLBJYtNX4ofZ3rX96R+eUEMznkQUe9BthYA0D+PJ4qe--RhYis1TnFa8dR8oy--Hod6JNYOfoJTx8SRqVrDAQ==
```

Comment:
add a new line at the end of the file to remove the github warning

Reply:
@asis-wtag bhai, If I add a new line, rails server is not starting. It gives some error.